### ISSUE TITLE
[MAS1.1.1] Alt is not descriptive for Images which appears after hit the 'Images on emulated slow network' button.

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-WebChat#1359]
**Actual Result:**
Alt is not descriptive for all 4 Images.
Screen reader focus is landing on ‘Decorative images' and reading as 'Microsoft surface'.

**Note:**
Same issue also reproducible for images of Layout control.

Expected Result:
Alt should be more descriptive for all 4 Images.

**Repro Steps:**

1. Open URL [https://microsoft.github.io/BotFramework-WebChat/full-bundle/](https://microsoft.github.io/BotFramework-WebChat/full-bundle/) in Edge browser.
2. Notice 'Upload File' and 'Send' button icons present in bottom of screen.
3. Type 'Help' in the input control at the bottom of the screen and hit enter (or click on send button).
4. Content will upload, Navigate to any "Images on emulated slow network" button by using Tab key and press Enter on it.
5. New data will upload.
6. Press F12 and open developer toolbar and check 'Alt' for images.
7. Observe that Alt is not descriptive for all 4 Images which appears after hit the 'Images on emulated slow network' button.

**User Impact:**
The screen reader user will face difficulty if screen reader announce same name for all decorative image.

**MAS Reference:**
[MAS 1.1.1 - Non-text Content (20)](https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={d2d2051f-bdc8-4af7-8e18-38aae867e216})

**Test Environment:**
**OS:** Windows 10
**OS Build:** 17134.345
**OS Version:** 1803
**Browser:** Edge

**Attachment:**
![mas1 1 1_alt is not descriptive for images](https://user-images.githubusercontent.com/41668935/48535575-d38f6880-e8d1-11e8-9c60-d7613c535992.png)

**Comment by compulim** [comment]
- [ ] Update MockBot to make the alt text descriptive
